### PR TITLE
Authentification A href VS button

### LINKED ISSUE #2138: Bogue - Le contrast de couleur ne fonctionne pas
## Description du bogue
En regardant l'exemple sur cette page : https://wet-boew.github.io/GCWeb/sites/authentication/contextual-signin-en.html 
Le bouton as été codé avec btn-primary. Mais quand ont le fait avec un btn-default le bouton n'est plus visible.

image
With the Hover state
image

## Exemple

Changé le btn-primary pour btn-default

## Comportement attendu
À ce que le contrast fonctionne pour tout les différents bouton.

## Ordinateur de bureau (veuillez compléter les informations suivantes) :
Tester avec Chrome 11.0.5563.111

### PR DESCRIPTION
**Pull request by crochefort**
PR pour cette issue : fix #2138  

La différence es au niveau que c'Est pas des <button> mais <a href avec la classe btn-default.

Il vas falloir corrigé la couleur pour possiblement les autres sortes de bouton

### CODE REVIEW

**Comment by duboisp** [comment]
@Garneauma can you generate and post a link to a working example for this PR? Thank you.

**Comment by Garneauma** [comment]
Demo link: https://servicecanada.github.io/wet-boew-demos/gcweb-pr2139/index.html

**Comment by duboisp** [comment]
DTO will take a quick look to ensure it is conform to their guidance.

Here a direct link to the newly added example: https://servicecanada.github.io/wet-boew-demos/gcweb-pr2139/sites/authentication/activeusersession-en.html

Which seems to be/will be used through a CDTS and leverage some icon for the session management.

cc @delisma @catalinaperezn 

---

Todo:

* [ ] (For wet-boew team) Add documentation to the authentication pattern to mention we do support an additional variant.

cc @GormFrank

**Comment by catalinaperezn** [comment]
We'll review it this week with the whole group at DTO.

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Comment by catalinaperezn** [comment]
Good morning friends. Sorry if I'm not doing this correctly. We are learning how to work together on different platforms. At DTO we reviewed the request and we have some questions we put in a google doc we are sending ... It's a draft but I wanted to share it fast instead of waiting for it to be perfect! We can iterate later... @duboisp @GormFrank  @delisma https://docs.google.com/document/d/1C0Ma7aasJ6iKl253nIbGuyO2J1ZOV4wl1az9_Vl15p8/edit?usp=sharing

**Comment by duboisp** [comment]
Will work on a way to present this demo pattern as a community variant and to find a way to illustrate that there is no official guidance approved that support this variant.

**Comment by duboisp** [comment]
Tomorrow at the office hours, PP and DTO would discuss on how we can merge this PR without creating false perception of "Official pattern" vs "Community pattern" vs "Trusted pattern" when presenting those working example.

**Comment by duboisp** [comment]
@Garneauma lorsque tu auras la chance, est ce que tu pourrais mettre à jour le démo temporaire? merci.

File: _includes/alert-community-stable.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{%- if page.language == "fr" -%}
+<div class="alert alert-info">
+	<p>Veuillez noter que la présente page web représente une contribution communautaire et qu'elle <strong>ne bénéficie pas</strong> de la validation ni de l'approbation formelle du bureau de la transformation numérique comparivement à une conception officielle. Il est à souligner que cette contribution ne va pas à l'encontre d'une version officielle publiée au moment de l'ajout. Il est possible que si une conception officielle équivalente est élaborée et soit formellement publiée que la conception démontré dans cette page en diffère en partie ou en totalité.</p>
```
Comment (GormFrank) [inline _includes/alert-community-stable.html:3]:
```suggestion
	<p>Veuillez noter que la présente page web représente une contribution communautaire et qu'elle <strong>ne bénéficie pas</strong> de la validation ni de l'approbation formelle du Bureau de la transformation numérique comparativement à une conception officielle. Il est à souligner que cette contribution ne va pas à l'encontre d'une version officielle publiée au moment de l'ajout. Si une conception officielle équivalente est élaborée et soit formellement publiée, il est possible que la conception démontrée sur la présente page en diffère en partie ou en totalité.</p>
```

File: _includes/alert-community-stable.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+	<p>Cette conception est <strong>stable</strong> et par conséquent elle bénificie des mêmes avantages que toutes autres fonctioanlité et conceptions stable tel que définie par <a href="https://wet-boew.github.io/wet-boew-documentation/decision/14.html" hreflang="en">l'ébauche de l'API publique de version de GCWeb</a>.</p>
+</div>
+{%- elsif page.language == "en" -%}
+<div class="alert alert-info">
+	<p>Please note that the current web page represents a community contribution and <strong>does not have</strong> formal validation or approval from the Digital Transformation Office compared to an official design. It should be noted that this contribution does not contradict any officially published version at the time of its addition. It is possible that when an equivalent official design is developed and get formally published, it may differ partially or entirely from the design shown on this page.</p>
+
+	<p>This <strong>stable</strong> design do share the same benefits as any other stable functionality and designs as defined by the <a href="https://wet-boew.github.io/wet-boew-documentation/decision/14.html">GCWeb public versioning API draft</a>.</p>
```
Comment (GormFrank) [inline _includes/alert-community-stable.html:11]:
```suggestion
	<p>This <strong>stable</strong> design do share the same benefits as any other stable functionalities and designs as defined by the <a href="https://wet-boew.github.io/wet-boew-documentation/decision/14.html">GCWeb public versioning API draft</a>.</p>
```

**Review by GormFrank — CHANGES_REQUESTED** [review]
Just a few spelling mistakes

**Comment by duboisp** [comment]
Pre-approved once the change is completed and a second review completed.

File: _includes/alert-community-stable.html (hunk 1 above)
Comment (duboisp) [reply _includes/alert-community-stable.html:3]: fixed

File: _includes/alert-community-stable.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{%- if page.language == "fr" -%}
+<div class="alert alert-info">
+	<p>Veuillez noter que la présente page web représente une contribution communautaire et qu'elle <strong>ne bénéficie pas</strong> de la validation ni de l'approbation formelle du bureau de la transformation numérique comparivement à une conception officielle. Il est à souligner que cette contribution ne va pas à l'encontre d'une version officielle publiée au moment de l'ajout. Il est possible que si une conception officielle équivalente est élaborée et soit formellement publiée que la conception démontré dans cette page en diffère en partie ou en totalité.</p>
+
+	<p>Cette conception est <strong>stable</strong> et par conséquent elle bénificie des mêmes avantages que toutes autres fonctioanlité et conceptions stable tel que définie par <a href="https://wet-boew.github.io/wet-boew-documentation/decision/14.html" hreflang="en">l'ébauche de l'API publique de version de GCWeb</a>.</p>
```
[the remark this answers is not in the document]
Comment (duboisp) [reply _includes/alert-community-stable.html:5]: fixed

File: _includes/alert-community-stable.html (hunk 2 above)
Comment (duboisp) [reply _includes/alert-community-stable.html:11]: fixed

File: sites/authentication/includes/authentication.html [hunk 4]
```diff
@@ -22,6 +22,9 @@ <h2 class="wb-inv">
 {%- elsif page.auth.type == "signedoff" -%}
 		<button class="btn btn-default" type="button">{{ i18nText-signOnRegister }}</button>
 		<button class="btn btn-primary" type="button">{{ i18nText-signIn }}</button>
+{%- elsif page.auth.type == "active" -%}
+		<p><a class="btn btn-sm btn-default" href="#"><span class="glyphicon glyphicon-user"></span> {{ i18nText-signUsername }}</a>
```
[the remark this answers is not in the document]
Comment (duboisp) [reply sites/authentication/includes/authentication.html:26]: We will apply a more global fix when we will officially assess and fix the glyphicon component to make it fully conform to WCAG 2.1

File: sites/authentication/includes/authentication.html [hunk 5]
```diff
@@ -22,6 +22,9 @@ <h2 class="wb-inv">
...
+		<a class="btn btn-sm btn-primary" href="#"><span class="glyphicon glyphicon-off"></span> {{ i18nText-signOut }}</a></p>
```
[the remark this answers is not in the document]
Comment (duboisp) [reply sites/authentication/includes/authentication.html:27]: We will apply a more global fix when we will officially assess and fix the glyphicon component to make it fully conform to WCAG 2.1

**Review by GormFrank — APPROVED** [review]
Solid first iteration of the Community stable alert box.

**Pull request merged**